eclass/ruby-ng.eclass

```diff
@@ -322,11 +318,9 @@ ruby_get_use_implementations() {
 ruby_get_use_targets() {
 	debug-print-function ${FUNCNAME} "${@}"
 
-	local t implementation
-	for implementation in $(_ruby_get_all_impls); do
-		t+=" ruby_targets_${implementation}"
-	done
-	echo $t
+
+	local impls="$(_ruby_get_all_impls)"
+	echo "${impls//ruby/ruby_targets_ruby}"
```

Comment:
Kinda hacky (assumes we won't ever support non `ruby*` targets) but sure.

Reply:
I'm not super worried about this but it's a fair point. I guess I can add a comment.